build.gradle

```diff
@@ -16,16 +6,17 @@ plugins {
     id 'net.ellune.blossom' version '1.0.1'
     id 'maven'
     id 'com.github.johnrengelman.shadow' version '1.2.3'
+    id "com.qixalite.spongestart" version "1.4.2"
+    id 'org.spongepowered.plugin' version '0.5.1'
```

Comment:
Why is this needed? Compiling against Sponge API 4+ does the mcmod.info file anyway.